### PR TITLE
Add layer-buttons for classic layer tree

### PR DESCRIPTION
**Pull request by hblitza**
## Feature
- Add settings dropdown button ( `LayerTreeDropDownContextMenu`)
- For layers of type `WMSTime`, add calendar button to adjust time interval (`LayerTreeApplyTimeInterval`)

image

What do you think @terrestris/devs ?

### CODE REVIEW

**Comment by hblitza** [comment]
Possible idea: Add prop like `showLayerButtons` that can be set within the classic tree class. To easily opt-out if the project does't need them

**Comment by dnlkoch** [comment]
Yes, I agree, the availability of both buttons should be configurable via a prop.

Otherwise :+1:

**Comment by weskamm** [comment]
> Possible idea: Add prop like `showLayerButtons` that can be set within the classic tree class. To easily opt-out if the project does't need them

+1 for that idea.

**Comment by hblitza** [comment]
Thanks for reviews. Prop added

File: src/component/container/LayerTreeClassic/LayerTreeClassic.tsx [hunk 1]
```diff
@@ -72,7 +78,28 @@ export class LayerTreeClassic extends React.Component<LayerTreeClassicProps> {
     } else {
       return (
         <div>
-          {layer.get('name')}
+          <div className="classic-tree-node-header">
+            <div>
+              {layer.get('name')}
+            </div>
+            {showLayerButtons &&
```
Comment (dnlkoch) [inline src/component/container/LayerTreeClassic/LayerTreeClassic.tsx:85]: The prop `showLayerButtons` is very unspecific. Please add a prop for each tree button, e.g. `showApplyTimeInterval` and `showContextMenu`.

**Review by dnlkoch — CHANGES_REQUESTED** [review]

File: src/component/container/LayerTreeClassic/LayerTreeClassic.tsx (hunk 1 above)
Comment (hblitza) [reply src/component/container/LayerTreeClassic/LayerTreeClassic.tsx]: Right!

**Pull request merged**